setup.py

```diff
@@ -22,6 +22,12 @@
     ],
     package_data={'tableau_utilities': ['tableau_file/*.yml']},
     include_package_data=True,
+    install_requires=['xmltodict>=0.12.0',
+                      'pyyaml>=6.0',
+                      'requests>=2.27.1',
+                      'pandas>=1.4.1',
+                      'tabulate>=0.8.9',
+                      'pytest>==7.1.2'],
```

Comment:
Can you also version bound these to within a major version number?
e.g. `xmltodict>=0.12.0,<1.0.0`

Think the biggest concern we'd have with people installing packages would be major updates.

Reply:
Also, `pytest` should not be included here.

Reply:
Done